recipes/buttons/callbar_button/callbar_button.vue

```diff
@@ -142,6 +143,16 @@ export default {
 
   },
 
+  emits: [
+    /**
+     * Native click event
+     *
+     * @event click
+     * @type {PointerEvent | KeyboardEvent}
+     */
+    'click',
+  ],
```

Comment:
If we remove this and the click event from `callbarButtonListeners()` and have `v-on="callbarButtonListeners"` on the button it should natively emit this event without us having to manually emit.

Reply:
Fair enough